### PR TITLE
fix failing integration test

### LINKED ISSUE #251: integrations tests failing at head
Integration tests are failing, e.g. https://github.com/cmu-delphi/delphi-epidata/runs/1260920716

```
 delphi.delphi-epidata.integrations.client.test_delphi_epidata.DelphiEpidataPythonClientTests.test_covidcast: fail
```

### PR DESCRIPTION
**Pull request by dfarrow0**
- see commit comments for details
- fixes #251

### CODE REVIEW

File: integrations/client/test_delphi_epidata.py [hunk 1]
```diff
@@ -38,69 +39,41 @@ def setUp(self):
...
-    self.maxDiff=None
 
     # insert dummy data
     self.cur.execute('''
       insert into covidcast values
         (0, 'src', 'sig', 'day', 'county', 20200414, '01234',
           123, 1.5, 2.5, 3.5, 456, 4, 20200414, 0, 0, False),
         (0, 'src', 'sig2', 'day', 'county', 20200414, '01234',
-          123, 1.5, 2.5, 3.5, 456, 4, 20200414, 0, 0, False),
+          123, 1.5, 2.5, 3.5, 456, 4, 20200414, 0, 1, False),
```
Comment (dfarrow0) [inline integrations/client/test_delphi_epidata.py:60]: this is the fix...

File: integrations/client/test_delphi_epidata.py [hunk 2]
```diff
@@ -38,69 +39,41 @@ def setUp(self):
...
         (0, 'src', 'sig', 'day', 'county', 20200414, '01234',
           456, 5.5, 1.2, 10.5, 789, 0, 20200415, 1, 0, False),
         (0, 'src', 'sig', 'day', 'county', 20200414, '01234',
           345, 6.5, 2.2, 11.5, 678, 0, 20200416, 2, 1, False)
     ''')
     self.cnx.commit()
 
-    # fetch data
```
Comment (dfarrow0) [inline integrations/client/test_delphi_epidata.py:64]: looks like git diff is having a really hard time with the lines below. there's no functional change, it's just:

- wrap each request/assert in a `subTest` for better output/debugging
- indent each by 2 (since `subTest` is a `with` block)
- wrap one row that was too long

File: integrations/client/test_delphi_epidata.py [hunk 3]
```diff
@@ -38,69 +39,41 @@ def setUp(self):
     # use the local instance of the Epidata API
     Epidata.BASE_URL = 'http://delphi_web_epidata/epidata/api.php'
 
+    # use the local instance of the epidata database
+    secrets.db.host = 'delphi_database_epidata'
+    secrets.db.epi = ('user', 'pass')
+
```
Comment (krivard) [inline integrations/client/test_delphi_epidata.py:45]: Why is this needed?

Comment (dfarrow0) [reply integrations/client/test_delphi_epidata.py:45]: The meta update test in this file fails without this addition *unless* some prior test happened to set it already. i.e. this makes it deterministic.

For example, this command fails:
```
docker run --rm --network delphi-net delphi_python python3 -m undefx.py3tester.py3tester --color repos/delphi/delphi-epidata/integrations/client
```

Whereas this command passes:
```
docker run --rm --network delphi-net delphi_python python3 -m undefx.py3tester.py3tester --color repos/delphi/delphi-epidata/integrations
```

The reason the latter passes is because some test in `integrations/acquisition` sets the database host/user/pass _before_ the meta update test in `integrations/client`.

**Review by krivard — APPROVED** [review]
xlnt
